examples/capi-quick-start/nutanix-cluster-calico-crs.yaml

```diff
@@ -0,0 +1,384 @@
+apiVersion: v1
+binaryData:
+  ca.crt: ${NUTANIX_ADDITIONAL_TRUST_BUNDLE=""}
+kind: ConfigMap
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: nutanix
+  name: ${CLUSTER_NAME}-pc-trusted-ca-bundle
+---
+apiVersion: v1
+data:
+  nutanix-ccm.yaml: |
```

Comment:
CCM should be deployed as an addon with `AfterControlPlaneInitialized` hook - happy to do this as follow up PR though.

Reply:
tracked thru https://d2iq.atlassian.net/browse/D2IQ-99841